File: SPX.Core/RNG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SPX.Core
{
    public class RNG
    {
        public static Random Rand;

        public static double Angle()
        {
            return RNG.Rand.Next(0, 360) * Math.PI / 180;
        }

        public static int Negative(int radius)
        {
            return RNG.Rand.Next(0, Math.Abs(radius) * 2) - Math.Abs(radius);
        }

        public static bool CoinFlip()
        {
            return RNG.Rand.Next(0, 2) == 1;
        }

        public static void Seed(int seed)
        {
            Rand = new Random(seed);
        }
    }
}

File: port-workspace/spx-ported/src/com/spx/core/RNG.java
package com.spx.core;import com.xna.wrapper.*;import java.util.*;
    public class RNG
    {
        public static Random Rand;
        public static int Next(int min,int max)        {        	return RNG.Rand.nextInt(max-min)+min;        }        
        public static double Angle()
        {
            return RNG.Rand.nextInt(360) * Math.PI / 180;
        }

        public static int Negative(int radius)
        {
            return RNG.Rand.nextInt(Math.abs(radius) * 2) - Math.abs(radius);
        }

        public static boolean CoinFlip()
        {
            return RNG.Rand.nextInt(2) == 1;
        }

        public static void Seed(int seed)
        {
            Rand = new Random(seed);            
        }
    }
